.github/workflows/e2e-test.yml

```diff
@@ -125,9 +128,12 @@ jobs:
       - name: Run E2E Tests
         run: npm run test:e2e
 
-      # Upload screenshots if any
-      - name: Upload screenshots
+      # Archive screenshots if any
+      - name: Archive e2e test screenshots
+        if: ${{ always() }}
```

Comment:
Just to confirm my understanding, we are using [always](https://docs.github.com/en/actions/learn-github-actions/expressions#always) here as this step is independent of previous steps and we'd always want to execute it.

Reply:
Yes, archiving screenshots should happen regardless of the previous steps' outcome. This is consistent with how we handle the same in other repos that handle the same, [exammple](https://github.com/woocommerce/woocommerce-admin/blob/main/.github/workflows/e2e.yml#L39).